dashboard.csss-ci.com/server/api/controllers/rest/v1/Complaints.php

```diff
@@ -93,7 +114,11 @@ public function town_get($id = null) {
             $this->response(['status' => FALSE, 'message' => $is_valid_token['message']], REST_Controller::HTTP_NOT_FOUND);
         }
     }
-
+    /**
+     * gets the id of the association of user from the complaint_model
```

Comment:
wrong

btw x1 ? x2 : x3 is shorthand for:
 if (x1) { 
     x2 
} else {
     x3 
}